stograde/specs/util.py

```diff
@@ -12,4 +15,20 @@ def check_dependencies(spec):
         try:
             os.stat(filepath)
         except FileNotFoundError:
-            warning('spec {}: required file "{}" could not be found'.format(spec['assignment'], filepath))
+            logging.warning('spec {}: required file "{}" could not be found'.format(spec['assignment'], filepath))
+
+
+def check_architecture(assignment, spec, ci):
+    # get check_architecture()
+    _, arch, _ = run(['uname', '-m'])
+    spec_arch = spec.get('architecture', None)
+    if not spec_arch or spec_arch == arch:
```

Comment:
Let’s be explicit here, and do “if spec_arch is None or spec_arch == arch”, please